pom.xml

```diff
@@ -1,64 +1,73 @@
+    </parent>
+    <groupId>pl.wedding.management.service</groupId>
+    <artifactId>wedding.management.service</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>wedding.management.service</name>
+    <description>Wedding Management Service</description>
 
-	<properties>
-		<java.version>15</java.version>
-	</properties>
+    <properties>
+        <java.version>15</java.version>
+    </properties>
 
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-jpa</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-validation</artifactId>
-		</dependency>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+        </dependency>
 
-		<dependency>
-			<groupId>org.projectlombok</groupId>
-			<artifactId>lombok</artifactId>
-			<optional>true</optional>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.junit.vintage</groupId>
-					<artifactId>junit-vintage-engine</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
-		<dependency>
-			<groupId>org.assertj</groupId>
-			<artifactId>assertj-core</artifactId>
-			<version>3.17.2</version>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.17.2</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        		<plugins>
+        			<plugin>
+        				<groupId>org.springframework.boot</groupId>
+        				<artifactId>spring-boot-maven-plugin</artifactId>
+        			</plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
This compiler plugin can be removed. It it sufficient to have  properties defined, as I mentioned man times, ex: https://github.com/wedding-hall-adm/wedding-management-service/pull/1#discussion_r505793348

Reply:
szukalem jak zdefiniowac te wlasciowosci ale iestety nie znalazlem i nie weim jak to zrobic

Reply:
I linked my comment, please read it.
You should remove whole <plugin> section with maven compiler. add properties in <properties> section, as described in my linked comment:
```
    <java.version>15</java.version>
    <maven.compiler.target>${java.version}</maven.compiler.target>
    <maven.compiler.source>${java.version}</maven.compiler.source>
```
In line #15 of this pom.xml, you already have: `java.version` property. Below it add those two for maven compiler.